lib/vyasa_web/components/contexts/read.ex

```diff
@@ -402,6 +402,31 @@ defmodule VyasaWeb.Context.Read do
     {:noreply, socket}
   end
 
+  @impl true
+  def handle_event(
+        "tombMark",
+        %{"id" => id},
+        %{
+          assigns: %{
+            kv_verses: verses,
+            marks: [%Mark{verse_id: v_id, binding: binding} | _] =  marks
+          }
+        } = socket
+      ) do
+
+    {
+      :noreply,
+      socket
+      |> assign(:marks, marks |> Enum.map(fn %{id: ^id} = m -> %{m | state: :tomb}
+      m -> m end))
+      |> mutate_draft_reflector()
+      |> stream_insert(
+        :verses,
+        %{verses[v_id] | binding: binding}
+      )
+    }
```

Comment:
why add this to the kv_verses map? is this redundant?

Reply:
hmm wat means? you mean the stream insertion?

Reply:
yeah

Reply:
actually its not the kv_verses its adding to 

:kv_verses map is being looked up so that the specific verse we are currently on (essentially coordinated by hover rune)  can be mutated and inserted back into the :verses stream
verse matrix. This ensures the mutation of our state of marks passes and is re-rendered at the DOM.

A brief primer on streams: 

Streams are handled on the client juxtapose against our more classical assigns which is handled on the server. 

Streams dont manage resources on the server end, so granular updates require targeting by id and since they don't remember things on server end we refer to our lookup :kv_verses map and  pass the entire update verse struct down the :verses stream.

Reply:
tldr; we are triggering the refresh cycle at the client side dom so our tombing or mutation of any marks state can be reflected at the targeted verse

Reply:
note to self:
so it's really just a way to trigger the dom diff because everything is relying on the state kept in the stream so stream update triggers the rerender